cypress/integration/app_spec.ts

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable jest/valid-expect, jest/expect-expect */
```

Comment:
Out of curiosity, is renaming this from `sample_spec.ts` mandatory to get Cypress working correctly?

Reply:
no, `sample_spec.ts` was the name of the initial boilerplate file that was generated when setting up cypress

Reply:
Ah, okay. Thanks for the follow-up!